experimental/dds/tree2/src/core/forest/forest.ts

```diff
@@ -99,7 +100,16 @@ export interface IForestSubscription extends Dependee, ISubscribable<ForestEvent
 	 * This dummy node can be used to read the detached fields,
 	 * but other operations (such as inspecting the dummy node's type or path) should not be relied upon.
 	 */
-	moveCursorToPath(destination: UpPath | undefined, cursorToMove: ITreeSubscriptionCursor): void;
+	moveCursorToPath(destination: UpPath, cursorToMove: ITreeSubscriptionCursor): void;
+
+	/**
+	 * The cursor is moved to a special dummy node above the detached fields.
+	 * This dummy node can be used to read the detached fields,
+	 * but other operations (such as inspecting the dummy node's type or path) should not be relied upon.
+	 * As this method does not return a forest cursor and cannot be freed, the caller of this method must
+	 * use with caution and ensure that it is not used incorrectly.
```

Comment:
We can't just tell people not to use it incorrectly. These docs are where we describe how you are allowed to use it. Also these limits are the same as the ones that apply to forest cursors, so its not because this isn't a forest cursors.
```suggestion
	 * While this method does not return an {@link ITreeSubscriptionCursor}, similar restrictions apply to its use:
	 * the returned cursor must not used after any edits are made to the forest.
```

Reply:
Updated comment with suggestion